src/core/storage/client.ts

```diff
@@ -17,12 +42,15 @@ export class StorachaClient implements StorageClient {
   private config: StorageConfig;
   private initialized: Promise<void> | null = null;
   private storage: Storage.Client | null = null;
+  /** Audience that will be used to verify the delegation */
+  private audience: Principal;
 
   constructor(config: StorageConfig) {
     this.config = {
       ...config,
       gatewayUrl: config.gatewayUrl || new URL(DEFAULT_GATEWAY_URL),
     };
+    this.audience = DID.parse(DEFAULT_AUDIENCE);
```

Comment:
We usually call this the `serviceID` or something like that as it is the _identity of the upload service_. In UCAN invocations the _audience_ field will be set to the service DID...

What is the idea with adding this as an instance property? Is it expected to be changed?

Reply:
It won't change. I had to set this because when I was creating the `serviceConf` it was saying the audience DID was not expected. It wanted `did:web:web3.storage` but got `did:web:up.storacha.network`. Then I defined this as part of the client attributes. But as you said, I need to rename it to `serviceId` instead of `audience`.